uPortal-portlets/src/main/java/org/apereo/portal/portlets/favorites/FavoritesController.java

```diff
@@ -87,11 +99,32 @@ public String initializeView(Model model) {
         // viable configured marketplace.  This complexity may not be worth it.  Anyway it is not yet implemented.
         model.addAttribute("marketplaceFname", this.marketplaceFName);
 
-        List<IUserLayoutNodeDescription> collections =
+        final List<IUserLayoutNodeDescription> collections =
                 FavoritesUtils.getFavoriteCollections(userLayout);
         model.addAttribute("collections", collections);
 
-        List<IUserLayoutNodeDescription> favorites = FavoritesUtils.getFavoritePortlets(userLayout);
+        final List<IUserLayoutNodeDescription> rawFavorites = FavoritesUtils.getFavoritePortlets(userLayout);
+
+        /*
+         * Filter the collection by SUBSCRIBE permission.  (NOTE:  I wish this filtering were done
+         * by the Layout Manager, before now, but it isn't.)
```

Comment:
I feel like I'm missing something here, if this belongs in the layout manager, why not add it to the layout manager?